Datahub/src/Datahub.Core/Model/Datahub/DatahubProjectDBContext.cs

```diff
@@ -178,5 +178,9 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
         modelBuilder.Entity<Project_Current_Monthly_Cost>()
             .Property(mc => mc.Id)
             .ValueGeneratedOnAdd();
+
+        modelBuilder.Entity<Datahub_Project_User>()
+            .Property(u => u.User_ID)
+            .ValueGeneratedOnAdd();
```

Comment:
We don't want to generate this value, this is the user's Azure GUID

Reply:
Hey @yjmrobert , sorry meant to make the value generated the ID for this table... Thanks for catching this